salt/modules/win_appx.py

```diff
@@ -0,0 +1,361 @@
+"""
+Manage provisioned apps
+=======================
+
+Provisioned apps are part of the image and are installed for every user the
```

Comment:
Can you add a versionadded for 3007 for this new module

Reply:
Done